pkg/operator/starter.go

```diff
@@ -79,10 +80,12 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 	if err != nil {
 		return err
 	}
+	configInformers := configinformers.NewSharedInformerFactory(configClient, 10*time.Minute)
```

Comment:
move to where other clients and informers are created.

don't forget to start it.

Reply:
Done.